gh-actions-scripts/cleanup_docker_images.sh

```diff
@@ -17,7 +17,6 @@ MAX_AGE_DAYS=30
 IMAGES=(
   "api"
   "bridge2"
-  "configuration-service"
```

Comment:
Please revert this. We still wanna clean up old config-svc images

Reply:
fixed, thx